services/api/src/resources/env-variables/resolvers.ts

```diff
@@ -187,7 +203,44 @@ const addEnvVariableToEnvironment = async (
   return R.prop(0, rows);
 };
 
-const deleteEnvVariable = async (
+// Stub/Example that implements add permission check
```

Comment:
whatever mutation ends up getting created for adding group env vars, the code in here should work